package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "oura-postgres-api",
+  "version": "1.0.0",
+  "description": "API for a Oura Postgres instance",
+  "type": "module",
+  "main": "app/index.js",
+  "scripts": {
+    "start": "node app/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dcSpark/oura-postgres-sink.git"
+  },
+  "author": "https://github.com/dcSpark",
+  "license": "ISC",
```

Comment:
Should be MIT / Apache 2

Reply:
oh I was going to ask about licensing actually, we need one on the entire repo